docs_user/modules/proc_preparing-controller-nodes-for-director-operator-adoption.adoc

```diff
@@ -0,0 +1,83 @@
+[id="preparing-controller-nodes-for-director-operator-adoption_{context}"]
+
+= Preparing Controller nodes for director Operator adoption
+
+Prepare the Controller nodes in your director Operator (OSPdO) environment for adoption to the {rhos_long} environment.
+
+[NOTE]
+====
+The following procedure uses a single {rhocp_long} cluster that is shared by the source OSPdO environment and a destination {rhos_acro} environment. Two {OpenShiftShort} master nodes are transferred to the {rhos_acro} environment, leaving OSPdO with one node for the duration of the adoption process. After adoption, the remaining node can be used by the {rhos_acro} environment as well.
+====
+
+.Procedure
+
+. Log in to the {OpenShiftShort} environment where {OpenStackShort} OSPdO is deployed and change to the project that hosts your {OpenStackShort} deployment:
++
+----
+$ oc project openstack
+----
+
+. Define the common environment variables:
++
+----
+export PASSWORD_FILE="tripleo-passwords.yaml"
+
+export RHOSO18_NAMESPACE="rhoso18"
+
+export OSPDO_NAMESPACE="openstack"
+
+export OS_CLIENT="oc exec -c openstackclient -t openstackclient -- "
+
+export CONTROLLER_SSH="oc rsh -n $OSPDO_NAMESPACE -c openstackclient openstackclient ssh controller-0.ctlplane"
+
+export CONTROLLER1_SSH="oc -n $OSPDO_NAMESPACE rsh -c openstackclient openstackclient ssh controller-0.ctlplane"
+export CONTROLLER2_SSH=
+export CONTROLLER3_SSH=
+
+export OSPDO_INTERNAL_API_NET="internalapi"
+
+export STORAGE_CLASS=$(oc get -n $OSPDO_NAMESPACE pvc openstackclient-hosts -o jsonpath='{.spec.storageClassName}')
+----
+
+. Get the name of the node where the current {OpenStackShort} Controller virtual machine (VM) is running:
++
+----
+$ export CONTROLLER_NODE=$(oc -n $OSPDO_NAMESPACE get vmi -ojson | jq -r '.items[0].status.nodeName')
+----
+
+. If you are using a combination master/worker cluster with a high availability OSPdO environment, configure OSPdO to use a single {OpenStackShort} Controller node:
+
+.. Disable stonith:
++
+----
+$ stonith_resources=$(sudo pcs stonith status|grep Started|awk '{print $2}')
+for in in stonith_resources ;do CONTROLLER1_SSH sudo pcs stonith disable ;done
```

Comment:
Just doublechecking that this is correct? Notice double "in" and space between resources and ;